package.json

```diff
@@ -44,5 +49,182 @@
   },
   "devDependencies": {
     "@cordova/eslint-config": "^3.0.0"
+  },
+  "homepage": "https://github.com/apache/cordova-plugin-media-capture#readme",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "dependencies": {
```

Comment:
What are all these new dependencies?

Reply:
@davidofwatkins ouch, Let me check that, it doesn't look correct